### ISSUE TITLE
Problems with automated setups / TYPO3-console when DB is empty

### ISSUE THREAD

**Issue by nabossha** [FluidTYPO3/flux#2072]
When setting up a fresh TYPO3-instance, i use the TYPO3-console to automate the process and also build & deploy to remote instances. This works well with flux 9.7.2 and LTS 11. 
Using `./vendor/bin/typo3cms install:setup --force --no-interaction` sets up an instance and creates the LocalConfiguration.php. Again, this works fine with 9.7.2

With the current dev-development releases of EXT:flux i get the following error during this step:
```
ERROR: 
An exception occurred while executing 'SELECT `uid` FROM `pages` WHERE
(`is_siteroot` = ?) AND (`sys_language_uid` = ?) AND (`t3ver_oid` = ?) AND
(`t3ver_wsid` = ?) AND ((`pages`.`deleted` = 0) AND (`pages`.`hidden` = 0))
ORDER BY `sorting` ASC' with params [1, 0, 0, 0]:

Table 'db.pages' doesn't exist

```
![Bildschirmfoto 2023-03-24 um 12 21 34](https://user-images.githubusercontent.com/3851681/227508346-61817e34-14f8-4f1d-b07b-f18623d575c5.png)

When i create the tables pages, sys_template and sys_registry manually **before** i run the setup, everything works again.
This seems very strange to me, that flux expects database-tables being available during setup-process. 
What is the reasons that flux checks for these tables?
At the moment i resolved the issue by creating the mentioned tables during a build-process manually, but this is very ugly and should not be required imho.

**Comment by NamelessCoder** [comment]
@nabossha Can you get me a stack backtrace (method calls alone is fine, e.g. `debug_print_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS | DEBUG_BACKTRACE_PROVIDE_OBJECT);`) for where this exception is raised?

**Comment by nabossha** [comment]
@NamelessCoder unfortunately, setting -vvv for the typo3cms-console commands does not give me any more useful output :-/

**Comment by NamelessCoder** [comment]
I'm afraid the only way to get a useful stack dump is to add the `debug_print_backtrace` in the source code of Doctrine where the exception is thrown. I _think_ the right place in this case is in the `catch` block in `\Doctrine\DBAL\Connection::executeQuery`.

PS: error tracing ability from the CLI is pretty horrible when the exception message is generic like this one...

**Comment by nabossha** [comment]
@NamelessCoder yes i notice the problem wich is kind of a chicken & egg stuff: when you setup a fresh instance, there is no database / tables available:

1. install TYPO3 via console <- this is where the error happens
2. setup Extensions via console
3. create / update database-scheme (which adds the tables from the EXTs)

**Comment by nabossha** [comment]
@NamelessCoder have you been able to look into the issue?
